### PR TITLE
[bitnami/redis] fix sentinel deployment

### LINKED ISSUE #9227: [bitnami/redis] latest is broken for a fresh deployment of redis-sentinel
### Name and Version

bitnami/redis 16.4.4

### What steps will reproduce the bug?

1. helm upgrade -i bitnami/redis --version 16.4.4 --set 'auth.enabled=false,sentinel.enabled=true'
2. observe `redis-node-0` never coming up

This is because a `timeout` has been removed from the script which ensures that the script can continue in this commit: https://github.com/bitnami/charts/commit/d6d66f1458cd7793691d4ea8d1c44da004f9b89d

### Are you using any custom parameters or values?

auth.enabled = false
sentinel.enabled=true

### What is the expected behavior?

A 3 node redis cluster to come online with sentinel working

### What do you see instead?

`INFO  ==> about to run the command: redis-cli -h redis.default.svc.cluster.local -p 26379 sentinel get-master-addr-by-name mymaster`

Will hang indefinitely and eventually the pod will be restarted due failing a liveness probe.

### Additional information

re-adding `timeout 5` to the relevant line fixes the issue, I've resorted to using `16.4.0` as all versions above this are currently broken. Removal of such a flag in a cli IMHO warrents a breaking change, not a "bugfix" version bump as it hides the meat of the work going on under the hood of the chart.

### PR DESCRIPTION
**Pull request by jack1902**
**Description of the change**

This change ensures that fresh deployments and existing deployments continue to function with redis sentinel mode. Previously a timeout existed which caused `redis-cli` get-master to timeout after 5 seconds but this caused a few issues. This PR addresses a problem introduced by removing this timeout, as probes never successfully triggered meaning the kubelet constantly restarted the pod before it managed to get up and running.

**Benefits**

Timeout is configurable for those who wish to reduce it, and startupProbes are enabled to ensure liveness/readiness aren't too eager to kill the pod when it hasn't had the opportunity to come up

**Possible drawbacks**

None that I can see, as this is setting the timeout to the default for the `redis-cli` to 220 whilst also ensuring that a bug that currently exists is fixed for those starting to use the redis chart first time with sentinel enabled.

**Applicable issues**

  - fixes #9227

**Additional information**

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/redis/README.md [hunk 1]
```diff
@@ -466,7 +467,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `sysctl.enabled`                                       | Enable init container to modify Kernel settings                                                 | `false`                 |
 | `sysctl.image.registry`                                | Bitnami Shell image registry                                                                    | `docker.io`             |
 | `sysctl.image.repository`                              | Bitnami Shell image repository                                                                  | `bitnami/bitnami-shell` |
-| `sysctl.image.tag`                                     | Bitnami Shell image tag (immutable tags are recommended)                                        | `10-debian-10-r351`     |
+| `sysctl.image.tag`                                     | Bitnami Shell image tag (immutable tags are recommended)                                        | `10-debian-10-r355`     |
```
Comment (jack1902) [inline bitnami/redis/README.md:470]: the tool used to update the images automatically is clearly skipping updates to the readme. I didn't change this but its something that is now included in my PR

**Comment by jack1902** [comment]
@alemorcuq PR opened to close the bug, CI won't complete as it requires approval

**Review by javsalgar — DISMISSED** [review]

**Comment by joao-luna-98** [comment]
This PR means breaking changes to clusters that do not support startup probes. Something to keep in consideration.

**Comment by rafariossaa** [comment]
Hi,
  Which k8s cluster are you running ?

**Comment by joao-luna-98** [comment]
> Hi, Which k8s cluster are you running ?

I'm running Kubernetes 1.11 on Openshift. (Something I can't change in the short term)

**Comment by rafariossaa** [comment]
Hi @joao-luna-98,
  Kubertenes 1.11 [ended its life](https://kubernetes.io/releases/patch-releases/) on 2019-05-01.
  I am sorry, but If you can not update, I would suggest to use local copy of the chart files and remove the probes or other unsupported features that may appear.

**Comment by joao-luna-98** [comment]
Disabling it is enough. I left a comment here for documentation purposes only. Thank you!

**Comment by rafariossaa** [comment]
Hi,
  Thanks.

**Pull request merged**